skyportal/plot.py

```diff
@@ -1588,7 +1763,8 @@ def spectroscopy_plot(obj_id, user, spec_id=None, width=600, device="browser"):
     )
 
     # Update the element spectral lines as well
-    for callback in callbacks:
+    for callback in callbacks:  
+        # it's all the callbacks and it's wroth for every one of them. should separate here also
```

Comment:
Not sure what this comment is supposed to mean.

Reply:
it was for my own understanding, can be removed